core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/VideoSizeChangedForwarder.java

```diff
@@ -0,0 +1,52 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import android.view.Surface;
+
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.decoder.DecoderCounters;
+import com.google.android.exoplayer2.video.VideoRendererEventListener;
+import com.novoda.noplayer.listeners.VideoSizeChangedListeners;
+
+class VideoSizeChangedForwarder implements VideoRendererEventListener {
```

Comment:
Same as others above. We no longer extend from `ExoPlayerEventForwarder` but instead implement the interface we care about directly from `ExoPlayer`.